src/v/kafka/server/handlers/sasl_authenticate.cc

```diff
@@ -20,7 +21,7 @@ ss::future<response_ptr> sasl_authenticate_handler::handle(
   request_context ctx, [[maybe_unused]] ss::smp_service_group g) {
     sasl_authenticate_request request;
     request.decode(ctx.reader(), ctx.header().version);
-    vlog(klog.debug, "Received SASL_AUTHENTICATE {}", request);
```

Comment:
this can probably be added back once the per-field redactions are available in the next commit

Reply:
Done